GW2EIEvtcParser/EIData/ProfHelpers/ProfHelper.cs

```diff
@@ -204,6 +213,13 @@ internal static class ProfHelper
         new MinionSpawnCastFinder(RelicPrivateerSpawn, (int)MinionID.TropicalBird)
             .UsingOrigin(InstantCastFinder.InstantCastOrigin.Gear)
             .WithBuilds(GW2Builds.November2024MountBalriorRelease),
+        new MissileCastFinder(RelicOfTheIce, RelicOfTheIce)
+            .UsingICD(1000)
+            .UsingOrigin(InstantCastFinder.InstantCastOrigin.Gear),
+        new MissileCastFinder(RelicOfPeithaBlade, RelicOfPeithaBlade)
+            .UsingOrigin(InstantCastFinder.InstantCastOrigin.Gear),
+        new MissileCastFinder(RelicOfDagdaHit, RelicOfDagdaHit)
```

Comment:
do we need this split when the buff based one is an accurate instant cast finder already?

Reply:
Removed it